colossalai/zero/sharded_model/sharded_model_v2.py

```diff
@@ -81,6 +83,10 @@ def backward(self, loss):
         loss.backward()
         self._final_backward_hook()
 
+    def backward_by_grad(self, tensor, grad):
```

Comment:
Why we have is API? Did PyTorch nn.Module has one?

Reply:
This API is in fact for pipeline parallel. The pipeline stage which is not the last stage has to backward with grad and tensor.